### PR TITLE
Release 1.2.0 articles

### PR DESCRIPTION
**Pull request by stefonarch**
I've  real bad internet connection atm, hopefully this evening it will recover.

### CODE REVIEW

**Comment by tsujan** [comment]
> I've real bad internet connection atm

Happy to know that I'm not alone ;) But I think my connection will be good enough for releasing LXQt tomorrow.

**Comment by stefonarch** [comment]
Connection is back, so after sunset I can finish the reviews of lxqt-themes Prs.

**Comment by stefonarch** [comment]
I'm done I guess, updated 3 screenshots to 1.2.0 on the website too (using gimp...). 
What about translations coming  in from this night?

**Comment by tsujan** [comment]
> What about translations coming in from this night?

No problem.

**Comment by tsujan** [comment]
But please use `lxqt-transupdate` to correct translations before tomorrow.

**Comment by stefonarch** [comment]
> But please use `lxqt-transupdate` to correct translations before tomorrow.

Sure, will do it now, hopefully no french coming in after...

File: _posts/2022-11-05-release-1.2.0.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+categories: release
+---
+
+The LXQt team announces the release of LXQt 1.2.0, the Lightweight Qt Desktop Environment.
+
+## Highlights
+
+### General
+
+ * Like its predecessor, LXQt 1.2.0 i still based on Qt 5.15, the last LTS version of Qt5.
```
Comment (tsujan) [inline _posts/2022-11-05-release-1.2.0.md:16]: LXQt 1.2.0 i still → LXQt 1.2.0 is still

It was my typo.

File: _posts/2022-11-05-release-1.2.0.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ * Preliminary changes have been made to LXQt Session for use under Wayland.
+ * Translations have received many updates.
+ * And other changes that can be found in change logs of LXQt components.
+
+### LibFM-Qt / PCManFM-Qt
+
+ * In the detailed list mode, items can be selected by dragging the mouse cursor inside non-name columns.
+ * `Ctrl+D` is added to PCManFM-Qt as well as LXQt file dialog for deselecting all items. It is especially useful in the detailed list mode.
+ * The search dialog entries are equipped with search history. The maximum number of history items can be set in Preferences → Advanced → Search.
+ * The margins of the desktop workspace can be customized. This cab be useful especially with panels/docks that do not reserve space but auto-hide on overlapping windows.
```
Comment (tsujan) [inline _posts/2022-11-05-release-1.2.0.md:30]: This cab be useful → This can be useful

My typo again.

**Pull request merged**